src/components/PlantsV2/components/HighestAndLowestMortalityRateSpeciesCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import strings from 'src/strings';
+import { Box, Divider, Typography, useTheme } from '@mui/material';
+import { ObservationResults } from 'src/types/Observations';
+import { selectSpecies } from 'src/redux/features/species/speciesSelectors';
+import { useAppSelector } from 'src/redux/store';
+
+type HighestAndLowestMortalityRateSpeciesCardProps = {
+  observation?: ObservationResults;
+};
+
+export default function HighestAndLowestMortalityRateSpeciesCard({
+  observation,
+}: HighestAndLowestMortalityRateSpeciesCardProps): JSX.Element {
+  const theme = useTheme();
+  const species = useAppSelector(selectSpecies);
+
+  let highestMortalityRate = 0;
+  let highestSpecies = '';
+
+  observation?.species.forEach((sp) => {
+    if (sp.mortalityRate && sp.mortalityRate > highestMortalityRate) {
+      highestMortalityRate = sp.mortalityRate;
+      // highestSpecies = sp.speciesName || '';
```

Comment:
Not sure why this speciesName is always null

Reply:
According to the api this field is:
```
If certainty is Other, the user-supplied name of the species. Null if certainty is Known or Unknown.
```
So it shouldn't be relied upon for a known species, i.e. one with an ID. The way your look it up below is correct.